infrastructure/lib/infrastructure-stack.ts

```diff
@@ -29,6 +30,19 @@ export class InfrastructureStack extends Stack {
     // Create VPC for the entire setup
     const vpcStack = new VpcStack(app, "OpenSearchHealth-VPC", {});
 
+    // Create secret related to GitHub Automation App
+    const openSearchMetricsGitHubAutomationAppSecretStack = new OpenSearchMetricsSecretsStack(app, "OpenSearchMetrics-GitHubAutomationApp-Secret", {
+      secretName: 'github-automation-app-creds'
```

Comment:
Nit: Suggest adding opensearch-project as part of the name.

Reply:
Updated @peterzhuamazon please check.